http_stubs/templatetags/stub_tags.py

```diff
@@ -16,3 +18,15 @@ def get_absolute_url_tag(context: Dict, url: Url) -> Url:
     :returns: absolute url
     """
     return context.get('request').build_absolute_uri(url)
+
+
+@register.filter(name='headers_to_list')
+@stringfilter
+def get_headers_list(headers: AnyStr, separator: AnyStr) -> List:
+    """Filter that creates a list from a string of headers(dict).
+
+    :param headers: string of headers
+    :param separator: separator to separate string
+    :returns: list of headers
```

Comment:
```suggestion
    :returns: list of individual headers string representation
```
(nit: unsure, but list of headers is too vague in my opinion).

Reply:
Changed. Thanks!